CloudFormation/arkcase.yml

```diff
@@ -113,12 +113,14 @@ Parameters:
       official AWS documentation here:
       https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html
     Default: db.t3.small
-    AllowedValues: [ db.m5.24xlarge, db.m5.16xlarge, db.m5.12xlarge, db.m5.8xlarge, db.m5.4xlarge, db.m5.4xlarge, db.m5.2xlarge, db.m5.xlarge, db.m5.large,
+    AllowedValues: [ db.m5.24xlarge, db.m5.16xlarge, db.m5.12xlarge, db.m5.8xlarge, db.m5.4xlarge, db.m5.4xlarge,
```

Comment:
I guess it's not possible to populate the instance types from some AWS configuration URL... they seem to update the instance types pretty often, is there any alternative to continuing to update this AllowedValues?  I seem to remember looking into this before a while ago.